packages/core/README.md

```diff
@@ -1,7 +1,17 @@
 # @contentful/experiences-core
 
-This package provides core internal functionality for the [Experiences SDK](https://www.contentful.com/developers/docs/experiences/set-up-experiences-sdk/).
+### Purpose
+- To contain shared code and utilities across the Experience-builder packages such as constants, types, transformers, and hooks.
```

Comment:
Is it currently "Experience-builder packages" or "Experiences packages"?

Reply:
`Experiences packages` is more correct